src/game/Render.cpp

```diff
@@ -22,6 +22,16 @@ void Game::render()
   renderNarrative();
 }
 
+void Game::renderEndScreen()
+{
+  refresh();
+  renderStatus();
+  colorWindow(developerWindow, RED_ON_BLACK);
+  string narrative = getNarrative();
+  mvwprintw(developerWindow, 3, 2, narrative.c_str());
+  wrefresh(developerWindow);
+}
+
```

Comment:
refreshes the status window so you can see your hearts are at 0
uses the developer window to display the narrative. This makes it so we can have other conditions for ending the game that display a message different from "GAME OVER: you died"